src/vs/workbench/contrib/debug/browser/debugService.ts

```diff
@@ -432,7 +433,23 @@ export class DebugService implements IDebugService {
 							nls.localize('debugTypeMissing', "Missing property 'type' for the chosen launch configuration.");
 					}
 
-					await this.showError(message);
+					let actionList: IAction[] = [];
+					let action = {
+						id: 'Install Debug Extension',
+						label: 'Install Debug Extension(s)',
```

Comment:
Whenever you have a user facing string make sure to use the `localize` method so the string gets translated.

Reply:
Great insight, thank you! Adjusted to `nls.localize('installAdditionalDebuggers', "Install {0} Extension", resolvedConfig.type),` let me know if that works.